minisign: add English and German pages (#5757)
tldr-pages/tldr: pages.de/common/minisign.md

# minisign

> > Ein einfaches und sicheres Werkzeug zum Signieren und Verifizieren von Dateien. Kompatibel mit `signify`.
> > Mehr Informationen: <https://jedisct1.github.io/minisign/>.

Reviewer: ```suggestion
> Ein einfaches und sicheres Werkzeug zum Signieren und Verifizieren von Dateien.
> Kompatibel mit `signify`.
> Mehr Informationen: <https://jedisct1.github.io/minisign/>.
```

- Ein neues Schlüsselpaar im Standardpfad generieren:

`minisign -G`

- Eine Datei signieren:

> `minisign -Sm {{datei}}`

Reviewer: ```suggestion
`minisign -Sm {{pfad/zu/datei}}`
```

- Eine Datei signieren und dabei einen vertrauenswürdigen (signierten) und einen nicht vertrauenswürdigen Kommentar in der Signatur anfügen:

> `minisign -Sm {{datei}} -c {{"Dieser Kommentar ist nicht vertrauenswürdig."}} -t {{"Dieser Kommentar ist vertrauenswürdig."}}`

Reviewer: ```suggestion
`minisign -Sm {{pfad/zu/datei}} -c "{{Dieser Kommentar ist nicht vertrauenswürdig.}}" -t "{{Dieser Kommentar ist vertrauenswürdig.}}"`
```

- Eine Datei und die vertrauenswürdigen Kommentare in ihrer Signatur gegen die angegebene Datei mit dem öffentlichen Schlüssel verifizieren:

> `minisign -Vm {{datei}} -p {{oeffentlicher_schluessel.pub}}`

Reviewer: ```suggestion
`minisign -Vm {{pfad/zu/datei}} -p {{oeffentlicher_schluessel.pub}}`
```

# minisign

> Ein denkbar einfaches Werkzeug um Dateien zu signieren und Signaturen zu verifizieren.
> Mehr Informationen: <https://jedisct1.github.io/minisign/>.

> - Ein neues Schlüsselpaar im Standardpfad generieren:

Reviewer 2: ```suggestion
- Generiere ein neues Schlüsselpaar im Standardpfad:
```

Reviewer 2: Did you resolve this on purpose?

Author: Oh, no. Forgot that one…

`minisign -G`

> - Eine Datei signieren:

Reviewer 2: ```suggestion
- Signiere eine Datei:
```

`minisign -Sm {{/pfad/zu/datei}}`

> - Eine Datei signieren und dabei einen vertrauenswürdigen (signierten) und einen nicht vertrauenswürdigen (unsignierten) Kommentar in der Signatur anfügen:

Reviewer: There's two of the same word (`vertrauenswürdigen`) here—is that intentional?  The English line contains `trusted` and `untrusted` 🤔

Author: Yes, this is intended. "Vertrauenswürdig" means "trusted" (or actually "trustworthy", but the literal translation for "trusted" would be "vertraut", which doesn't really make sense in German in this context), "nicht" means "not", thus, the latter is "not trusted". I *could* translate it literally as "unvertrauenswürdig", however, I think this would be considered somewhat poor style. Thanks for pointing this out anyway, though.

Reviewer: Ah, I see! Thank you for this in-depth explanation! ❤️

> - Eine Datei signieren und dabei einen vertrauenswürdigen (signierten) und einen nicht vertrauenswürdigen (unsignierten) Kommentar in der Signatur anfügen:

Reviewer 2: ```suggestion
- Signiere eine Datei und füge dabei einen vertrauenswürdigen (signierten) und einen nicht vertrauenswürdigen (unsignierten) Kommentar in der Signatur an:
```

> `minisign -Sm {{/pfad/zu/datei}} -c "{{Dieser Kommentar ist nicht vertrauenswürdig.}}" -t "{{Dieser Kommentar ist vertrauenswürdig.}}"`

Reviewer 2: ```suggestion
`minisign -Sm {{/pfad/zu/datei}} -c "{{Nicht vertrauenswürdiger Kommentar}}" -t "{{Vertrauenswürdiger Kommentar}}"`
```
The shorter, the better, except you don't like it.

> - Eine Datei und die vertrauenswürdigen Kommentare in ihrer Signatur gegen die angegebene Datei mit dem öffentlichen Schlüssel verifizieren:

Reviewer 2: ```suggestion
- Verifiziere eine Datei und die vertrauenswürdigen Kommentare in ihrer Signatur gegen die angegebene Datei mit dem öffentlichen Schlüssel:
```

> `minisign -Vm {{pfad/zu/datei}} -p {{pfad/zu/oeffentlicher_schluessel.pub}}`

Reviewer 2: ```suggestion
`minisign -Vm {{pfad/zu/datei}} -p {{pfad/zu/öffentlicher_schlüssel.pub}}`
```

> - Eine Datei und die vertrauenswürdigen Kommentare in ihrer Signatur gegen den angegebenen, in Base64 kodierten öffentlichen Schlüssel verifizieren:

Reviewer 2: ```suggestion
- Verifiziere eine Datei und die vertrauenswürdigen Kommentare in ihrer Signatur gegen den angegebenen, in Base64 codierten öffentlichen Schlüssel:
```
`kodiert` doesn't actually exist. (Just found out myself)

> `minisign -Vm {{pfad/zu/datei}} -P "{{oeffentlicher_schluessel_base64}}"`

Reviewer 2: ```suggestion
`minisign -Vm {{pfad/zu/datei}} -P "{{öffentlicher_schlüssel_base64}}"`
```
